app/league/config_utils.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""Helper utility for managing application settings."""
+import json
+from os.path import exists, getsize, join
+
+from flask import current_app
+
+
+def update_config_file(new_config):
```

Comment:
While this works, I really don't think it's a great idea. Once we start moving state outside the database, we lose guarantees of persistence if/when we add things like backups, failover copies, etc. I think we should try to do this with SQLAlchemy. If you'd like to discuss this on Slack or on a call I'm happy to